TSN/README.md

```diff
@@ -0,0 +1,28 @@
+# TSN
+
+This repo provides one method for Person Action Recognition task.
+
+## Get started
+Prerequisites
+pip3 install -r requirements.txt
+
+## Data preparation
+We use Kinetics400 to train and evaluate our models. Please refer to (https://github.com/open-mmlab/mmaction/tree/master/data_tools/kinetics400) for data prepration.
```

Comment:
We train and evaluate our models on Kinetics400 dataset. Please refer to (https://github.com/open-mmlab/mmaction/tree/master/data_tools/kinetics400) for data prepration.